packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -787,13 +787,35 @@ const DateTimePicker = ({
     setAbsoluteValue(newAbsolute);
   };
 
+  // Validate that the start time if less than the end time
+  const validateStartEndTime = (startTime, endTime) => {
+    const timefrom = new Date();
+    let temp = startTime.split(':');
+    timefrom.setHours(Number(temp[0]));
+    timefrom.setMinutes(Number(temp[1]));
+
+    const timeto = new Date();
```

Comment:
You'll need to pass the start/end dates from the absolute range picker into this method, too. By comparing only against the current date with `new Date()` you're showing any `startTime` that's after `endTime` as invalid, but that's not always true. If I have chosen `11/1/2021` as my start date, and `11/3/2021` as my end date. I can have whatever startTime/endTime I want and all of them are valid.

Reply:
Thanks for letting me know to include the start and end date. I've taken that into consideration and refactored the code.